int32 id
float32 force
float32 position
bool in_contact
bool in_error
bool[8] errors
string[] error_messages
